spec/features/govuk_verify_spec.rb

```diff
@@ -53,6 +53,48 @@
       @claim.reload
       expect(@claim.full_name).to eql("Isambard Kingdom Brunel")
     end
+
+    context "with unspecified gender parameter" do
```

Comment:
I think we have this coverage already on `missing_verify_information_spec.rb`?

Reply:
ah yes, I missed that!